### PR TITLE
Make foreman_proxy tftp_servername configurable

### PR DESCRIPTION
**Pull request by bdurrow**

### CODE REVIEW

File: ansible/roles/foreman_proxy/templates/tftp.yml.j2 [hunk 1]
```diff
@@ -8,3 +8,7 @@
 
 # Defines the TFTP Servername to use, overrides the name in the subnet declaration
 #:tftp_servername: tftp.domain.com
+
+{% if foreman_proxy_tftp_host is defined %}
+:tftp_servername: {{ foreman_proxy_tftp_host }}
```
Comment (karras) [inline ansible/roles/foreman_proxy/templates/tftp.yml.j2:13]: Should we change the variable name to `foreman_proxy_tftp_servername` thus it's more aligned with the corresponding Foreman setting?

**Review by karras — CHANGES_REQUESTED** [review]
Can you also please expose the variable in the defaults but comment / deactivate it?

File: ansible/roles/foreman_proxy/templates/tftp.yml.j2 (hunk 1 above)
Comment (bdurrow) [reply ansible/roles/foreman_proxy/templates/tftp.yml.j2:13]: I agree that foreman_proxy_tftp_host was a poor choice.  I could see using foreman_proxy_tftp_servername but I am of the opinion that the variables should be more consistent across ansible than sync'd with the application ansible is configuring.  I see foreman_proxy_dhcp_server is already used so I could see changing it to foreman_proxy_tftp_server also.  Do you have a strong preference?

Comment (karras) [reply ansible/roles/foreman_proxy/templates/tftp.yml.j2:13]: `foreman_proxy_tftp_server` sounds good to me. :)

**Comment by karras** [comment]
Project archived

**Pull request closed without merging**